meson.build

```diff
@@ -428,12 +436,18 @@ if get_option('devkits').length() != 0
   endif
   mkdevkit = [python, releng / 'mkdevkit.py']
   uninstalled_dir = meson.global_build_root() / 'meson-uninstalled'
+
+  prefix_syms = get_option('prefixsyms')
```

Comment:
It would be good to invert the logic, so it's opt-out instead. In that way we won't have to update frida-core, which also uses `mkdevkit.py`.